Makefile

```diff
@@ -28,5 +28,7 @@ install: venv
 	@echo Installed
 
 venv:
+	$(SYSTEM_PYTHON) -m pip install --upgrade pip
+	$(SYSTEM_PYTHON) -m pip install virtualenv
```

Comment:
remove

Reply:
Without this, you can't use the install script